llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -4700,33 +4700,65 @@ SDValue AArch64TargetLowering::LowerMGATHER(SDValue Op,
                                MGT->getMemOperand(), IndexType, ExtType);
   }
 
+  // Lower fixed length gather to a scalable equivalent.
```

Comment:
These changes should not be here

Reply:
yes, sorry thats my fault. I guess when i did git cherry-pick, this code added in my commit